secretservice/secretservice.go

```diff
@@ -73,6 +76,18 @@ func NewService() (*SecretService, error) {
 	}
 	signalCh := make(chan *dbus.Signal, 16)
 	conn.Signal(signalCh)
+
+	// It is necessary to add signal match for broadcast secret prompt
+	// signals (without destination) to be received by session. Otherwise
+	// some secret service providers will not work properly e.g. KeepassXC.
+	o, ok := conn.BusObject().(*dbus.Object)
+	if !ok {
+		return nil, errors.New("bus object type not supported")
+	}
+	if err := o.AddMatchSignal("org.freedesktop.Secret.Prompt", "Completed").Store(); err != nil {
```

Comment:
what do you think of the change in https://github.com/keybase/go-keychain/pull/85/files? will this cover your usecase?